.github/workflows/rsyslog.yml

```diff
@@ -125,3 +130,27 @@ jobs:
           cd rsyslog
           devtools/gather-check-logs.sh
           cat failed-tests.log
+
+      - name: rsyslog test run in centos 8 container
+        run: |
+          cd rsyslog
+	  make clean
```

Comment:
you need to replace the tab by spaces, else YML does not validate. That's the reason the test does not start. It's unfortunately a silent fail.

Reply:
as a further side-note, if you want to test via a container, you need to install the new librelp inside that container. The current testbench avoids this by using the VM itself as testbed.

Reply:
yes i see, had to fuix this already multiple times ;)

Reply:
I do not to run it in a container. But the problem is the error is happening on centos 8 or fedora 30 only so far. 
So I am stuck now ...

Reply:
well... than it would probably be best to build and install librelp inside the container. That, of course, needs some extra plumbing. Or you do a docker build beforehand - that might be a solution. Are you already on a path?